### PR TITLE
docs(README) : update README

### PR DESCRIPTION
**Pull request by siddanand**
- Add link to the main repo
- Add instructions for setup & running the project 
- Remove learn more and deploy to Vercel sections
- Add a contribution guidelines section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,29 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Babitz Frontend
 
-## Getting Started
+This is a [Next.js](https://nextjs.org/) project.
```
Comment (chirgjn) [inline README.md:3]:
```suggestion
This is a [Next.js](https://nextjs.org/) project. If you want to know more about Babitz, you can read the checkout the [main project repo](https://github.com/pesto-students/babitz)).
```

File: README.md [hunk 2]
```diff
@@ -18,17 +40,10 @@ You can start editing the page by modifying `pages/index.js`. The page auto-upda
...
+---
 
-## Deploy on Vercel
+## Want to contribute?
 
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+Before proceeding further with this project kindly go through all the documentation provided at [Babitz Main Repo](https://github.com/pesto-students/babitz).
 
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+Also refer to **[Contribution guideline](/CONTRIBUTING.md)** there you will find all the necessary information regarding the how to create file, write and structure code for this project feel free to contribute if you think we miss on any guideline that should be followed.
```
Comment (chirgjn) [inline README.md:49]:
```suggestion
Please refer to **[Contribution guideline](/CONTRIBUTING.md)**. You will find all the necessary information regarding the how to create file, write and structure code for this project. Feel free to contribute, suggestion for changes in the guidelines are also welcomed.
```

**Review by chirgjn — CHANGES_REQUESTED** [review]

**Review by chirgjn — APPROVED** [review]

**Comment by chirgjn** [comment]
@siddanand can you add a comment for the final commit message here in the required format

**Comment by siddanand** [comment]
Add link to the main repo

**Comment by chirgjn** [comment]
> @siddanand can you add a comment for the final commit message here in the required format

@siddanand

**Comment by siddanand** [comment]
> > @siddanand can you add a comment for the final commit message here in the required format
> 
> @siddanand

Added link to the main repo

**Comment by siddanand** [comment]
docs: add link to main repo

**Comment by siddanand** [comment]
### docs : update README 
- Add link to the main repo
- Add to setup and run project
- Remove learn more and deploy to vercel
- Add want to contribute section

**Comment by siddanand** [comment]
### docs : update README
- Add link to the main repo
- Add how to setup and run project
- Remove learn more and deploy to Vercel
- Add want to contribute section

**Pull request merged**